app.py

```diff
@@ -496,10 +497,24 @@ def sync():
     SYNCApi().download_new_mails_from_server()
 
 
+@eel.expose
+def pong():
+    return "ping"
+
+
+@eel.expose
+def log(message):
+    logger.debug(f'UI: {message}')
+
+
+@eel.expose
+def log_error(message):
+    logger.error(f'UI: {message}')
+
+
```

Comment:
Possiamo rinominarle in ui_log e ui_log_error? Altrimenti rischiamo di fare confusione!

Reply:
Fatto